src/System.Private.CoreLib/Resources/Strings.resx

```diff
@@ -2428,7 +2428,10 @@
   <data name="InvalidOperation_CannotImportGlobalFromDifferentModule" xml:space="preserve">
     <value>Unable to import a global method or field from a different module.</value>
   </data>
-  <data name="InvalidOperation_CannotRemoveLastFromEmptyCollection" xml:space="preserve">
+  <data name="InvalidOperation_CannotRegisterSecondResolver" xml:space="preserve">
+    <value>A Resolver is alerady Set for the Assembly.</value>
```

Comment:
Type: alerady

Reply:
Typo: Set in the middle of the sentence should be lower case.

Reply:
Fixed it, thanks.